contracts/EpochTokenLocker.sol

```diff
@@ -4,34 +4,29 @@ import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
 import "openzeppelin-solidity/contracts/math/Math.sol";
 import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 
-
 // EpochTokenLocker saveguards tokens for applications with constant-balances during discrete epochs
 // It allows to deposit token which become credited in the next epoch and allows to request a token-withdraw
 // which becomes claimable after the current epoch expired.
 contract EpochTokenLocker {
-    using SafeMath for uint;
+    using SafeMath for uint256;
 
     event Deposit(
         address user,
         address token,
-        uint amount,
-        uint stateIndex
+        uint256 amount,
```

Comment:
I like this change. We should be explicit about width where ever we can

Reply:
I am less excited about this, but its alright I guess...